examples/java/core/serialization/src/main/bond/serialization.bond

```diff
@@ -0,0 +1,8 @@
+namespace examples.serialization
```

Comment:
I agree with your point. I lean towards using the namespace mapping rather than putting the Java-oriented namespace in the .bond file, since the former facilitates sharing the files across languages a bit more naturally.